packages/zod/src/v4/core/tests/locales/he.test.ts

```diff
@@ -0,0 +1,370 @@
+      }
+    });
+  });
+
+  describe("invalid_format with string checks", () => {
+    test("startsWith", () => {
+      const schema = z.string().startsWith("hello");
+      const result = schema.safeParse("world");
+      expect(result.success).toBe(false);
+      if (!result.success) {
+        expect(result.error.issues[0].message).toBe('המחרוזת חייבת להתחיל ב"hello"');
+      }
+    });
+
+    test("endsWith", () => {
+      const schema = z.string().endsWith("world");
+      const result = schema.safeParse("hello");
+      expect(result.success).toBe(false);
+      if (!result.success) {
+        expect(result.error.issues[0].message).toBe('המחרוזת חייבת להסתיים ב"world"');
+      }
+    });
+
+    test("includes", () => {
+      const schema = z.string().includes("test");
+      const result = schema.safeParse("hello world");
+      expect(result.success).toBe(false);
+      if (!result.success) {
+        expect(result.error.issues[0].message).toBe('המחרוזת חייבת לכלול "test"');
+      }
+    });
+
+    test("regex", () => {
+      const schema = z.string().regex(/^[a-z]+$/);
+      const result = schema.safeParse("ABC123");
+      expect(result.success).toBe(false);
+      if (!result.success) {
+        expect(result.error.issues[0].message).toBe("המחרוזת חייבת להתאים לתבנית /^[a-z]+$/");
+      }
+    });
+  });
+
+  describe("invalid_format with common formats", () => {
+    test("email", () => {
+      const schema = z.string().email();
+      const result = schema.safeParse("not-an-email");
+      expect(result.success).toBe(false);
+      if (!result.success) {
+        expect(result.error.issues[0].message).toBe("כתובת אימייל לא תקין");
+      }
+    });
+
+    test("url", () => {
+      const schema = z.string().url();
+      const result = schema.safeParse("not-a-url");
+      expect(result.success).toBe(false);
+      if (!result.success) {
+        expect(result.error.issues[0].message).toBe("כתובת רשת לא תקין");
+      }
+    });
+
+    test("uuid", () => {
+      const schema = z.string().uuid();
+      const result = schema.safeParse("not-a-uuid");
+      expect(result.success).toBe(false);
+      if (!result.success) {
+        expect(result.error.issues[0].message).toBe("UUID לא תקין");
+      }
+    });
+  });
+
+  describe("tuple validation", () => {
+    test("invalid element type in tuple", () => {
+      const schema = z.tuple([z.string(), z.number()]);
+      const result = schema.safeParse(["abc", "not a number"]);
+      expect(result.success).toBe(false);
+      if (!result.success) {
+        expect(result.error.issues[0].message).toContain("המספר צריך להיות");
+      }
+    });
+  });
+
+  describe("inclusive vs exclusive bounds", () => {
+    test("inclusive minimum (>=)", () => {
+      const schema = z.number().min(10);
+      const result = schema.safeParse(5);
+      expect(result.success).toBe(false);
+      if (!result.success) {
+        expect(result.error.issues[0].message).toContain(">=10");
```

Comment:
i would check the whole message when possible, here and others, since it's much easier to check the "hebrew vailidity" of it